console/command_in_controller.rst

```diff
@@ -20,11 +20,9 @@ Instead, you can run the command directly from the controller.
     a controller has a slight performance impact because of the request stack
     overhead.
 
-Imagine you want to send spooled Swift Mailer messages by
-:doc:`using the swiftmailer:spool:send command </email>`.
-Run this command from inside your controller via::
+Imagine you want to run the `debug:twig` from inside your controller::
```

Comment:
The example is not great, but frankly, I think we should remove this doc as you should never have to do this.